hardware/electrical/power_supply.kicad_sch

```diff
@@ -5,6 +5,2544 @@
+    (property "Intersheet References" "${INTERSHEET_REFS}" (id 0) (at 227.3359 38.1794 0)
+      (effects (font (size 1.27 1.27)) (justify right) hide)
+    )
+  )
+  (global_label "LIPO+" (shape input) (at 149.86 45.72 0) (fields_autoplaced)
+    (effects (font (size 1.27 1.27)) (justify left))
+    (uuid e537a8dd-28f3-43ee-9aaf-ab81b3bbfe9a)
+    (property "Intersheet References" "${INTERSHEET_REFS}" (id 0) (at 158.0788 45.6406 0)
+      (effects (font (size 1.27 1.27)) (justify left) hide)
+    )
+  )
+
+  (symbol (lib_id "Transistor_FET:IRLZ34N") (at 116.84 64.77 90) (mirror x) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid 12d05c8d-0164-461d-9f03-ec024d0a1ebc)
+    (property "Reference" "Q2" (id 0) (at 116.84 71.12 90))
+    (property "Value" "IRLZ34N" (id 1) (at 116.84 73.66 90))
+    (property "Footprint" "Package_TO_SOT_SMD:SOT-23" (id 2) (at 118.745 71.12 0)
+      (effects (font (size 1.27 1.27) italic) (justify left) hide)
+    )
+    (property "Datasheet" "http://www.infineon.com/dgdl/irlz34npbf.pdf?fileId=5546d462533600a40153567206892720" (id 3) (at 116.84 64.77 0)
+      (effects (font (size 1.27 1.27)) (justify left) hide)
+    )
+    (pin "1" (uuid b495b080-2737-47e7-bba7-0faf092538c2))
+    (pin "2" (uuid 4f06eb0d-c7b5-43ee-9d36-b1aff211ce49))
+    (pin "3" (uuid 5c5a3677-bba7-4b00-a759-9aa8269e9bc0))
+  )
+
+  (symbol (lib_id "Device:R") (at 167.64 39.37 180) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid 266b237c-74ad-49be-bef3-c1c7ef100fb5)
+    (property "Reference" "R6" (id 0) (at 165.1 38.1 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Value" "1k" (id 1) (at 165.1 40.64 0)
+      (effects (font (size 1.27 1.27)) (justify left))
+    )
+    (property "Footprint" "Resistor_SMD:R_0805_2012Metric_Pad1.20x1.40mm_HandSolder" (id 2) (at 169.418 39.37 90)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "~" (id 3) (at 167.64 39.37 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid 1dad003d-f9cc-455b-bff3-1e744fc3f5d0))
+    (pin "2" (uuid 282b938e-ca26-4766-ab0e-c5f77e171738))
+  )
+
+  (symbol (lib_id "power:VBUS") (at 38.1 38.1 0) (unit 1)
+    (in_bom yes) (on_board yes) (fields_autoplaced)
+    (uuid 280a0024-fc5d-4c37-87c1-a11c2f0d3e28)
+    (property "Reference" "#PWR0104" (id 0) (at 38.1 41.91 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Value" "VBUS" (id 1) (at 38.1 33.02 0))
+    (property "Footprint" "" (id 2) (at 38.1 38.1 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (property "Datasheet" "" (id 3) (at 38.1 38.1 0)
+      (effects (font (size 1.27 1.27)) hide)
+    )
+    (pin "1" (uuid facbd4fd-8fa5-4e53-9de2-88d64b80dda2))
+  )
+
+  (symbol (lib_id "Switch:SW_DIP_x01") (at 191.77 50.8 0) (unit 1)
+    (in_bom yes) (on_board yes)
+    (uuid 2e6b50c0-de46-4ba8-a4c3-a06e87bb17e2)
+    (property "Reference" "SW1" (id 0) (at 191.77 46.99 0))
```

Comment:
Do we need this switch? Switches can be quite large and expensive so could we omit it or replace it with a solder jumper?

Reply:
The switch is meant for the user to switch on/off the dash cam.

Reply:
Aren't we going to be using the microcontroller's sleep modes rather than a hard power-off so we can preserve the RTC time? If that's the case I think the power button will just be hooked up to a GPIO pin.

Reply:
SW1 removed in cd04ce8. User power button will be added as a GPIO input to the MCU at a later date.

Reply:
Power button added as MCU GPIO input in d5d6c5ddbc67e39fed1ce54bdeec3217946e603a.